src/api/middleware/validators/offer.js

```diff
@@ -508,13 +508,14 @@ const canBeManaged = async (req, res, next) => {
 
     // Admin or gods can enable even if it was blocked by another admin
     if ((!req.user?.isAdmin && req.body.god_token !== config.god_token) &&
-         offer.hiddenReason === OfferConstants.HiddenOfferReasons.ADMIN_BLOCK) {
+        offer.hiddenReason === OfferConstants.HiddenOfferReasons.ADMIN_BLOCK) {
         return next(new APIError(HTTPStatus.FORBIDDEN, ErrorTypes.FORBIDDEN, ValidationReasons.OFFER_BLOCKED_ADMIN));
     }
 
     return next();
 };
 
+// Validator to check if the offer can be enabled
```

Comment:
Why this comment? I think the function name is pretty self-explanatory

Reply:
The rest of the functions had a describing comment to go with them, so i thought it would be better to also add it here

Reply:
Remove the comment. It adds nothing to the function name